### ISSUE TITLE
[a11y] dialog navigation issue using shift+tab

### ISSUE THREAD

**Issue by Etat-Plateforme** [react-bootstrap/react-bootstrap#1193]
According to http://www.w3.org/TR/wai-aria-practices/#dialog_modal, an aria-labelledby property must be set on the dialog, pointing to an element acting as a label for the dialog. Here is an example:
```<div class="modal-dialog" aria-labelledby="modal-title">
	<h4 id=”modal-title” class=”modal-title”>Title</h4>
</div>```

If no element can act as a label, a simple aria-label can also be used.

When the dialog is shown, focus must be trapped inside it, so the user cannot focus an element outside of it. It works fine with tab but not when using shift + tab.

**Comment by jquense** [comment]
Hi!

You can and should be adding the aria-labels, yourself, since there is no way we can pick a label for you. In terms of Shift-Tab, it does work, except when you reach the first focusable item in the modal, rather then switch to the last item it stays at the first. This is unfortunately, a technical limititaion at this time. But you can raise and issue on the react-overlays repo, which implements all the modal focus logic.
